stroom-pipeline/src/main/java/stroom/pipeline/server/writer/FileAppender.java

```diff
@@ -132,6 +140,9 @@ public void setOutputPaths(final String outputPaths) {
         this.outputPaths = outputPaths.split(",");
     }
 
+    @PipelineProperty(description = "Apply GZIP compression to output files", defaultValue = "false")
+    public void setUseCompression(final boolean useCompression) { this.useCompression = useCompression; }
```

Comment:
public void setUseCompression(final boolean useCompression) {
        this.useCompression = useCompression;
    }